v1.1-beta/matrixx-digital-commerce/README.md

```diff
@@ -0,0 +1,59 @@
+# MATRIXX Digital Commerce
+
+You will need the images loaded on your system. This test uses versions 5270 ,5261 and 5241 in some cases for rollback.
+The main version being tested is 5270.
+
+
+## Set Up
+
+Set up the testsuite like normal
+
+```
+cnf-testsuite setup
+```
+
```

Comment:
@deerowbear please add some text saying to go into each sub-directory and run these commands. 

Otherwise please add READMEs to each sub-directory and link from the top level.  Something for your end-users to be able to recreate the results if desired.

Reply:
@taylor updated the directories we run the commands from in helm project.